docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -127,25 +101,20 @@ Finally, during the data plane adoption, you will have to take care of several a
         fixedIP: 192.168.122.100
         name: ctlplane
         subnetName: subnet1
-```
-
-- expand SSH range for the firewall configuration to include both subnets:
+----
 
-```yaml
+. Expand the SSH range for the firewall configuration to include both subnets:
++
+----
         edpm_sshd_allowed_ranges:
         - 192.168.122.0/24
         - 172.31.0.0/24
-```
-
-This is to allow SSH access from the new subnet to the adopted nodes as well as
-the old one.
-
----
-
-Since you are applying new network configuration to the nodes, consider also
-setting `edpm_network_config_update: true` to enforce the changes.
+----
++
+This provides SSH access from the new subnet to the adopted nodes as well as
+the old nodes.
 
----
+. Optional: Set `edpm_network_config_update: true` to enforce the changes that you are applying to the nodes:
 
 Note that the examples above are incomplete and should be incorporated into
```

Comment:
I’m not sure what purpose this serves or what we are trying to say. Are we trying to say that the examples don’t necessarily reflect the customers’ configuration and that they need to use values that are specific to their configuration?

Reply:
Yes, this was the intent. I guess it's redundant, if we assume that readers are careful. One can only hope! (But if they are not, this note won't help much either.) I'm ok if we just remove it.

Reply:
I think we can keep it, but it should be moved to the beginning of the procedure and rewritten, e.g.,:

"The values that are used in the following procedure are examples. Use values that are specific to your configuration."